packages/react-menu/src/components/MenuItemRadio/MenuItemRadio.test.tsx

```diff
@@ -44,10 +44,10 @@ describe('MenuItemRadio', () => {
 });
 
 describe('MenuItemRadio', () => {
-  const TestMenuListContext = (props: { children: React.ReactNode; context?: Partial<MenuListContext> }) => {
-    const contextValue: MenuListContext = {
+  const TestMenuListContext = (props: { children: React.ReactNode; context?: Partial<MenuListContextValue> }) => {
```

Comment:
nit: The same naming proposal as in https://github.com/microsoft/fluentui/pull/17113/files#r580924669.